Client/Frontend/Browser/TabManager.swift

```diff
@@ -358,6 +360,51 @@ class TabManager : NSObject {
     func removeAllPrivateTabsAndNotify(notify: Bool) {
         privateTabs.forEach({ removeTab($0, flushToDisk: true, notify: notify) })
     }
+    
+    func removeTabsWithUndoToast(tabs: [Tab]) {
+        tempTabs = tabs
+        var _tabs = tabs
```

Comment:
Nit: I would refrain from using underscores to indicate local vars. It's more of a C/C++ convention. Why not `tabsCopy`?